anta/tests/routing/bgp.py

```diff
@@ -469,15 +467,29 @@ class Input(AntaTest.Input):
         def validate_bgp_peers(cls, bgp_peers: list[BgpPeer]) -> list[BgpPeer]:
             """Validate that 'advertised_routes' or 'received_routes' field is provided in each BGP peer."""
             for peer in bgp_peers:
-                if peer.advertised_routes is None or peer.received_routes is None:
-                    msg = f"{peer} 'advertised_routes' or 'received_routes' field missing in the input"
+                if peer.advertised_routes is None and peer.received_routes is None:
+                    msg = f"{peer} 'advertised_routes' or 'received_routes' must be provided"
```

Comment:
We already have that convention elsewhere (BGP tests).
```suggestion
                    msg = f"{peer} 'advertised_routes' or 'received_routes' field missing in the input"
```